terraform/testcases/xrayreceiver_mock/parameters.tfvars

```diff
@@ -0,0 +1,4 @@
+validation_config="default-mocked-server-xrayreceiver-validation.yml"
+
+# sample application image to emit the trace data
+data_emitter_image="johnwu20/sample_server_app:v2.0.0"
```

Comment:
just to make sure we maintain all the sample app in one repo?

Reply:
yes we will, this is a temp one